assets/i18n/en.json

```diff
@@ -164,6 +164,7 @@
     "I18N_GET_STARTED_PAGE_PARAGRAPH_9": "When learners go through your exploration, they can send you feedback to alert you to problems or to share ideas for making it better.",
     "I18N_GET_STARTED_PAGE_PARAGRAPH_9_HEADING": "Improve Your Exploration",
     "I18N_GET_STARTED_PAGE_TITLE": "Get Started",
+    "I18N_INTERACTIONS_FRACTIONS_INPUT_PLACEHOLDER": "Enter a fraction in the form n/n or n n/n.",
```

Comment:
Instead of n/n, maybe a/b or a b/c might be more descriptive. Just a thought.

Reply:
I was considering something like that, but I think some of the younger learners may not understand that (especially if they don't have algebra experience yet).

Reply:
Oh, ok, fine then.